src/com_tjvendors/site/helpers/mails.php

```diff
@@ -60,13 +60,20 @@ public function onAfterVendorCreate($vendorDetails)
 	{
 		$adminEmailArray = array();
 		$adminEmail = (!empty($this->tjvendorsparams->get('email'))) ? $this->tjvendorsparams->get('email') : $this->siteConfig->get('mailfrom');
-		$adminEmailArray = explode(',', $adminEmail);	
+
+		$adminEmailArray = explode(',', $adminEmail);
```

Comment:
do we need to chk if this is not empty?

$this->siteConfig->get('mailfrom');
=> does not seem to be good way to get admin email and hence admin user id

Reply:
code updated